ui/raidboss/data/02-arr/trial/titan-nm.txt

```diff
@@ -0,0 +1,105 @@
+### Titan Story / Normal
+# -ii 5C5 3CD -p 28B:203.0  28C:603.0
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+### Phase 1 100->85%
+0.0 "Start"
+0.0 "--sync--" sync /Engage!/ window 0,1
+2.0 "--sync--" sync /:Titan:368:/ window 2,0
+5.0 "Rock Buster" sync /:Titan:281:/
+7.0 "Tumult" sync /:Titan:282:/
+16.1 "Tumult" sync /:Titan:282:/
+19.2 "Rock Buster" sync /:Titan:281:/
+25.2 "Tumult" sync /:Titan:282:/ window 8,8
+
+32.3 "Rock Buster" sync /:Titan:281:/
+34.3 "Tumult" sync /:Titan:282:/
+43.4 "Tumult" sync /:Titan:282:/
+46.5 "Rock Buster" sync /:Titan:281:/
+52.5 "Tumult" sync /:Titan:282:/ window 8,8 jump 25.2
+
+59.6 "Rock Buster"
+61.6 "Tumult"
+70.7 "Tumult"
+73.8 "Rock Buster"
+79.8 "Tumult"
+
+### Phase 2 85->55%
+200.0 "--sync--" sync /:28B:Titan/ window 200,0
+203.0 "Geocrush" sync /:Titan:28B:/
+
+211.6 "Landslide" sync /:Titan:28A:/ window 211,17
+217.8 "Rock Buster" sync /:Titan:281:/
+221.8 "Tumult" sync /:Titan:282:/
+229.1 "Landslide" sync /:Titan:28A:/
+235.3 "Rock Buster" sync /:Titan:281:/
+239.3 "Tumult" sync /:Titan:282:/ window 17,16
+
+246.6 "Landslide" sync /:Titan:28A:/
+252.8 "Rock Buster" sync /:Titan:281:/
+256.8 "Tumult" sync /:Titan:282:/ window 15,17
+264.1 "Landslide" sync /:Titan:28A:/
+270.2 "Rock Buster" sync /:Titan:281:/
+274.2 "Tumult" sync /:Titan:282:/ window 17,16 jump 239.3
+
+281.5 "Landslide"
+287.6 "Rock Buster"
+291.6 "Tumult"
+298.9 "Landslide"
+305.1 "Rock Buster"
+309.1 "Tumult"
+
+### Phase 3 Heart Phase (55%)
+400.0 "--sync--" sync /:28B:Titan/ window 196,0
+403.0 "Geocrush" sync /:Titan:28B:/
+413.6 "Rock Throw" sync /:Titan:285:/
+421.5 "Landslide" sync /:Titan:28A:/
+427.6 "Rock Buster" sync /:Titan:281:/ window 20,15
+431.6 "Tumult" sync /:Titan:282:/
+435.1 "Rock Throw" sync /:Titan:285:/
+442.9 "Landslide" sync /:Titan:28A:/
+449.1 "Rock Buster" sync /:Titan:281:/
+453.1 "Tumult" sync /:Titan:282:/ window 16,25
+464.6 "Landslide" sync /:Titan:28A:/
+473.6 "Enrage"
+
+###Phase 4 55->0%
+601.9 "--sync--" sync /:28C:Earthen Fury/ window 199,0 
+603.0 "Earthen Fury" sync /:Titan:28C:/

```

Comment:
This has a 2.1 second cast time?

Is there a reason not to pick a whole number here, like `600.0` instead of `601.9`?

Reply:
Titan lands and starts casting Earthen Fury for 1.1s then it goes off. This was the only way I had something to link to that wasn't an RP line.

Reply:
Ah sorry, apparently I can't math.  1.1s just seemed unusually fast to me.  It's fine to sync to that, was mostly just curious if it were truly 1.1s and also why you picked 601.9 or 603 as a time here.

Would you also mind syncing a one-time ability phase like this back to the beginning with a `window 610,0` or something? I usually try to put something a couple of those in a fight to make sure things are on track, just in case.

Reply:
I think what quisquous is asking is "why does this phase start at 601.9 instead of 600?"  It's mostly a stylistic question, but unless there's a very good reason for it, you are probably better off adjusting this loop back by 1.9 seconds to match the rest of the repository. (Or you might adjust it back 3.8 seconds, so that the Earthen Fury cast lines up at 600.)